src/Addon/SettingsPage.php

```diff
@@ -0,0 +1,342 @@
+<?php
+namespace GiveAddon\Addon;
+
+/**
+ * Example code to show how to add setting page to give settings.
+ *
+ * @package     GiveAddon\Addon
+ * @subpackage  Classes/Give_BP_Admin_Settings
+ * @copyright   Copyright (c) 2020, GiveWP
+ */
+class SettingsPage extends \Give_Settings_Page {
+
+	/**
+	 * Settings constructor.
+	 */
+	public function __construct() {
+		$this->id          = 'give-addon-boilerplate';
```

Comment:
@JasonTheAdams do think it will be a good idea to set setting page slug with a bash script?

Reply:
That might be a good idea. Additionally, we can use that value to rename the give-addon-boilerplate.php entry file to something meaningful.